spec/helpers/stylesheet.js

```diff
@@ -14,12 +17,27 @@ describe('stylesheet helper', () => {
     const runTestCases = testRunner({siteSettings});
 
     it('should render a link tag with the cdn ulr and stencil-stylesheet data tag', done => {
-        runTestCases([
-            {
-                input: '{{{stylesheet "assets/css/style.css"}}}',
-                output: '<link data-stencil-stylesheet href="https://cdn.bcapp/hash/stencil/123/css/style-xyz.css" rel="stylesheet">',
-            },
-        ], done);
+
+        const template = '{{{stylesheet "assets/css/style.css" resourceHint="preload"}}}';
+        const expected = '<link data-stencil-stylesheet href="https://cdn.bcapp/hash/stencil/123/css/style-xyz.css" rel="stylesheet">';
+
+        const renderer = buildRenderer(siteSettings);
```

Comment:
I think those test cases could be written in less code. 
1. testRunner on line 17 accepts `renderer` as a param. 
so you could wrap it with `beforeEach(() => {})`, so each time there would be a fresh copy of the `renderer`.
2. 2nd param of the `runTestCases()`, is a callback function, which is run at the end of the rendering (all promises). 
[Check this out](https://github.com/bigcommerce/paper-handlebars/blob/master/spec/spec-helpers.js#L47)
So you could add some your custom callbacks over there and after those at the end call `done();`